consensus/pacemaker/debug.go

```diff
@@ -0,0 +1,28 @@
+package pacemaker
+
+import typesCons "github.com/pokt-network/pocket/consensus/types"
+
```

Comment:
Can you add an annonymous initialisation? It's how we enforce that a struct complies with an interface.

Should look something like this:
`var _ PacemakerDebug = &pacemakerDebug`

Reply:
noted, thanks.